core/modules/qdisp/testQDisp.cc

```diff
@@ -0,0 +1,412 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2015 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include <string>
+#include <unistd.h>
+
+// Boost unit test header
+#define BOOST_TEST_MODULE Qdisp_1
+#include "boost/test/included/unit_test.hpp"
+
+// External headers
+#include "boost/chrono.hpp"
+#include "boost/chrono/duration.hpp"
+#include "boost/make_shared.hpp"
+
+// LSST headers
+#include "lsst/log/Log.h"
+
+// Qserv headers
+#include "ccontrol/MergingRequester.h"
+#include "css/Facade.h"
+#include "global/ResourceUnit.h"
+#include "global/MsgReceiver.h"
+#include "qdisp/MessageStore.h"
+#include "qdisp/XrdSsiMocks.h"
+#include "util/threadSafe.h"
+
+// Quick and easy way to access private members for testing.
+#define private public
+#include "qdisp/Executive.h"
+#undef private
+
+namespace test = boost::test_tools;
+using namespace lsst::qserv;
+
+typedef util::Sequential<int> SequentialInt;
+
+struct TestFixture {
+    TestFixture(void) {
+        // To learn how to dump the map, see qserv/core/css/KvInterfaceImplMem.cc
```

Comment:
Comment here not relevant to test and could be reaped.  Actually, the whole test fixture is not used and could be reaped -- then just change BOOST_FIXTURE_TEST_SUITE to BOOST_TEST_SUITE below?